### ISSUE TITLE
Dropdowns causing accessibility test failures

### ISSUE THREAD

**Issue by Blackbaud-ScottFreeman** [blackbaud/skyux-popovers#65]
### Expected behavior
Dropdowns would pass accessibility tests.

### Actual behavior
Dropdowns are failing accessibility tests.  Example output:
```
    Failed: Expected element to pass accessibility checks.
    aXe - [Rule: 'aria-valid-attr-value'] ARIA attributes must conform to valid values - WCAG: wcag131, wcag411, wcag412
           Get help at: https://dequeuniversity.com/rules/axe/2.6/aria-valid-attr-value?application=axeAPI
           Elements:
    <button _ngcontent-a-c2="" aria-haspopup="true" class="sky-btn sky-dropdown-button sky-dropdown-button-type-select sky-btn-default" type="button" ng-reflect-klass="sky-btn sky-dropdown-button" ng-reflect-ng-class="sky-dropdown-button-type-selec" aria-expanded="false" aria-controls="sky-dropdown-menu-1" aria-label="Context menu">
    <button _ngcontent-a-c2="" aria-haspopup="true" class="sky-btn sky-dropdown-button sky-dropdown-button-type-select sky-btn-default" type="button" ng-reflect-klass="sky-btn sky-dropdown-button" ng-reflect-ng-class="sky-dropdown-button-type-selec" aria-expanded="false" aria-controls="sky-dropdown-menu-2" aria-label="Select color value">
    <button _ngcontent-a-c2="" aria-haspopup="true" class="sky-btn sky-dropdown-button sky-dropdown-button-type-select sky-btn-default" type="button" ng-reflect-klass="sky-btn sky-dropdown-button" ng-reflect-ng-class="sky-dropdown-button-type-selec" aria-expanded="false" aria-controls="sky-dropdown-menu-3" aria-label="Select color value">
```

### Steps to reproduce
Create an accessibility test against a component with a dropdown.

### Severity
Low

### Impact
Low.  SPAs with dropdowns and accessibility tests will not be able to upgrade their SPAs using the recommended way of running `skyux upgrade`.
